src/transaction.rs

```diff
@@ -14,12 +20,11 @@ use rkyv::{Archive, Deserialize, Serialize};
 
 use dusk_bls12_381::BlsScalar;
 use dusk_bytes::{DeserializableSlice, Error as BytesError, Serializable};
-use dusk_poseidon::cipher::PoseidonCipher;
 
-use crate::{Crossover, Fee, Message, Note};
+use crate::{Crossover, Fee, Note};
 
-const STCO_MESSAGE_SIZE: usize = 7 + 2 * PoseidonCipher::cipher_size();
-const STCT_MESSAGE_SIZE: usize = 5 + PoseidonCipher::cipher_size();
+/// Type alias for the ID of a module.
+pub type ModuleId = [u8; 32];
```

Comment:
There's another example of this in this very crate already, and that is a `dusk_plonk::Proof`. The `proof` field of a transaction really _should be_ that a `Proof`, however, in the interest of reducing dependencies `dusk_plonk` hasn't been included in the manifest.
In fact, this example is even worse, since to convert back and forth between `Proof` and `Vec<u8>` is not `const`, while converting between `[u8; 32]` and `piecrust_uplink::ModuleId` is.
I suggest an issue be created on how to deal with this.

Reply:
Created issue https://github.com/dusk-network/phoenix-core/issues/121 to deal with this.